src/main/app/fees/feesClient.ts

```diff
@@ -45,39 +43,37 @@ export class FeesClient {
    * @param amount amount in pounds
    * @returns {Promise.<CalculationOutcome>} promise containing the calculation outcome (including fee amount in pennies)
    */
-  static calculateFee (feeCode: string, amount: number): Promise<CalculationOutcome> {
+  static async calculateFee (feeCode: string, amount: number): Promise<CalculationOutcome> {
     ClaimValidator.claimAmount(amount)
     const amountInPennies = MoneyConverter.convertPoundsToPennies(amount)
     if (amountInPennies <= 0) {
       throw new Error(`Amount must be at least 1 penny, amount was: ${amountInPennies}`)
     }
 
-    return request.get(`${feesUrl}/range-groups/${feeCode}/calculations?value=${amountInPennies}`)
-      .then((body: Object) => plainToClass(CalculationOutcome, body))
+    const fee: object = await request.get(`${feesUrl}/range-groups/${feeCode}/calculations?value=${amountInPennies}`)
+    return plainToClass(CalculationOutcome, fee)
   }
+
   /**
    * Get the issue fee range group
    * @returns {Promise.<RangeGroup>} promise containing the range group (including fee amounts in pennies)
    */
-  static getIssueFeeRangeGroup (): Promise<RangeGroup> {
+  static async getIssueFeeRangeGroup (): Promise<RangeGroup> {
     return this.getRangeGroup(issueFeeCode)
   }
 
   /**
    * Get hearing fee range group
    * @returns {Promise.<RangeGroup>} promise containing the range group (including fee amounts in pennies)
    */
-  static getHearingFeeRangeGroup (): Promise<RangeGroup> {
+  static async getHearingFeeRangeGroup (): Promise<RangeGroup> {
     return this.getRangeGroup(hearingFeeCode)
   }
 
-  private static getRangeGroup (code: string): Promise<RangeGroup> {
+  private static async getRangeGroup (code: string): Promise<RangeGroup> {
     if (StringUtils.isBlank(code)) {
       throw new Error('Fee code is required')
     }
-    return request.get(`${feesUrl}/range-groups/${code}`)
-      .then((body: Object) => {
-        return plainToClass(RangeGroup, body)
-      })
+    return plainToClass(RangeGroup, await request.get(`${feesUrl}/range-groups/${code}`) as object)
```

Comment:
I would recommend to extract response as local variable.

Reply:
done